File: AlHadoop/src/main/java/com/al/util/AllAnalysis.java
package com.al.util;

import com.al.hive.HiveAnalysis;
import com.al.mapreduce.TimeRun;

public class AllAnalysis {

    public static void main(String[] args) {
        HiveAnalysis.runAnalysis();
        TimeRun.runAnalysis();
    }

}

File: AlSpark/src/main/scala/com/al/util/AllAnalysis.scala
package com.al.util

import com.al.spark.core._
import com.al.spark.mllib.{ChannelAnalysis, GenderAnalysis}

/**
  * Created by An on 2016/11/30.
  */
object AllAnalysis {

  def main(args: Array[String]): Unit = {
    FlowAnalysis.runAnalysis()
    SearchAnalysis.runAnalysis()
    ProvinceAnalysis.runAnalysis()
    CountryAnalysis.runAnalysis()
    ContentAnalysis.runAnalysis()

    GenderAnalysis.runAnalysis()
    ChannelAnalysis.runAnalysis()
  }

}